src/settings-list.js

```diff
@@ -429,5 +429,11 @@ module.exports = [
                 chat.helpers.serverMessage("Chat scrollback is now set to: " + lines, true);
             }
         }
+    },
+    {
+        name: 'Right click user to chat',
+        description: 'When clicking a chat line it\'ll copy the senders username to input box.',
```

Comment:
"Double-clicking a username in chat copies it into the chat text box" would be more appropriate.